examples/postprocess_dag.json

```diff
@@ -55,15 +58,33 @@
       "additional_attributes": {
         "nt_db": "s3://idseq-database/20170824/blast_db/nt"
       }
+    },
+    {
+      "in": ["fastqs", "amr_db"],
+      "out": "srst2_out",
+      "class": "PipelineStepRunSRST2",
+      "module": "idseq_dag.steps.run_srst2",
+      "additional_files": {"resist_gene_db": "s3://idseq-database/test/AMR/ARGannot_r2.fasta"},
+      "additional_attributes": {
+        "min_cov": 0,
+        "n_threads": 16,
+        "output_prefix": "output",
```

Comment:
Our convention is that the local output directory is specified in the engine, so you would remove `output_prefix` from the dag specification and just use `self.output_dir_local` as the output_prefix argument to SRST2.